static/app/views/organizationGroupDetails/groupMerged/mergedItem.tsx

```diff
@@ -117,16 +125,38 @@ class MergedItem extends React.Component<Props, State> {
 
         {!collapsed && (
           <MergedEventList className="event-list">
-            {event && (
+            {latestEvent && (
               <EventDetails className="event-details">
                 <EventOrGroupHeader
-                  data={event}
+                  data={latestEvent}
                   organization={organization}
                   hideIcons
                   hideLevel
                 />
               </EventDetails>
             )}
+
+            {children && (
```

Comment:
can `children` be an empty array?

```suggestion
            {!!children?.length && (
```

Reply:
thanks, I forget that `[]` is not false